src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.json

```diff
@@ -12,7 +12,8 @@
       "description": ".NET MAUI SDK for Mobile",
       "extends": [ 
         "maui-android",
-        "maui-ios"
+        "maui-ios",
+        "maui-tizen"
```

Comment:
I don't think we can put `maui-tizen` by default here. Can you remove this, and customers would have to install `maui-tizen` specifically for Tizen support?

Reply:
OK. I'll remove it and check if there is any problem through build test.